include/eve/module/real/core/function/regular/simd/x86/popcount.hpp

```diff
@@ -127,4 +128,90 @@ namespace eve::detail
       }
     }
   }
+
+  /////////////////////////////////////////////////////////////////////////////
+  // avx512
+  template<integral_scalar_value T,  typename N>
+  EVE_FORCEINLINE auto popcount_(EVE_SUPPORTS(avx512_), wide<T, N>  v) noexcept
+      requires x86_abi<abi_t<T, N>>
+  {
+    auto constexpr bitalg = spy::supports::avx512::bitalg_;
+    auto constexpr ppctdq = spy::supports::avx512::popcntdq_;
```

Comment:
but we don't have this on CI.
If we don't have something on CI, adding it is a problem.

Reply:
ok I just wanted to see if it worked on CI, the PR was the to see if there were problems.
On my computer I have bitalg and popcntdq and I wanted to see if it was easy to add these 
At home it works enabling the options on the compiler and also just putting -march=skylake...

Reply:
What I mean is.

By avx512 we mean:
`AVX512 with skylake like minimum flavour (f,vl,dq,bw)`

https://jfalcou.github.io/eve/index.html

I'd prefer if we either stick by it or compile versions where it's disabled on CI as well.

Otherwise we have a code that's never tested, which means that the users might hit it and get runtime bugs.

Reply:
ok I finish testing my stuffs and will remove the bitalg etc part,
but it is not never tested as it was tested on my computer who have it

Reply:
I meant on ci. The code needs to be contiguous tested so that the new changes don't break old behaviour.